apm-hub-helm/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for apm-hub.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+deployment:
+  label: apm-hub
+
+image:
+  repository: docker.io/flanksource/apm-hub
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "apm-hub"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "apm-hub-sa"
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 8080
+  targetPort: 8080
+  protocol: TCP
+
+configYaml: |
+  backends:
+    - kubernetes:
+        kubeconfig:
+
+ingress:
```

Comment:
I don't actually see an ingress being created ?

Reply:
I've not created an ingress as I didn't see it in the repo. Do we want the ingress to be created as part of the helm chart? 
If not I'll remove the unused parts of the values file since they are not needed.

Reply:
Added ingress.